src/openapi/streaming/serializer/serializer-protobuf.js

```diff
@@ -0,0 +1,125 @@
+import * as baseProtos from './schemas/proto-base';
+import protobuf from 'protobufjs/dist/protobuf';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import base64 from '@protobufjs/base64';
```

Comment:
namespaced export from protobufjs. Not sure why it's detected as an error by eslint. By using built in base64 from protobufjs we save some space by not using separate base64 implementation.

Reply:
Not needed anymore. In latest version, util is available in protobuf.